src/mixin/entity-mixin-lit.js

```diff
@@ -18,60 +18,27 @@ export const EntityMixinLit = superclass => class extends superclass {
 			/**
 			 * Entity object that extends the Entity class.
 			 */
-			_entity: { type: Object },
+			entity: { type: Object }
 		};
 	}
 
-	constructor() {
-		super();
-	}
-
-	set href(href) {
```

Comment:
A lot of these custom property setters seemed superfluous as they are just copying the default lit implementation, so I removed them.

Reply:
So these were not superfluous. They made it so updateRequests were not called. Sorry just noticed this now. It was an optimization and never changed the resulting execution. I should have put a comment.